perfkitbenchmarker/linux_benchmarks/cloudsuite_websearch_benchmark.py

```diff
@@ -107,7 +110,8 @@ def _CheckHeapSize(heap_size_str):
 def GetConfig(user_config):
   config = configs.LoadConfig(BENCHMARK_CONFIG, user_config, BENCHMARK_NAME)
   num_clients = FLAGS.cs_websearch_num_clients
-  config['vm_groups']['default']['vm_count'] = num_clients + NUM_SERVERS
+  config['vm_groups']['workers']['vm_count'] = NUM_SERVERS
```

Comment:
Doesn't look like we need the "NUM_SERVERS" constant anymore now that there are separate groups.